src/main/java/frc/robot/commands/ClimbingCommands/SetSpeedRightMotorClimbing.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.ClimbingCommands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.ClimbingSubsystem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class SetSpeedRightMotorClimbing extends InstantCommand {
+  private final ClimbingSubsystem rightMotorClimbing = ClimbingSubsystem.getInstance();
```

Comment:
change `rightMotorClimbing` to `climbingSubsystem`

Reply:
fixed